modules/test/playwright/tests/users-admin-web/main/users.spec.ts

```diff
@@ -246,3 +252,216 @@ test(
+		const user2 = await apiHelpers.headlessAdminUser.postUserAccount();
+		const user3 = await apiHelpers.headlessAdminUser.postUserAccount();
+
+		await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+			account.id,
+			[user1.emailAddress]
+		);
+		await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+			organization.id,
+			user2.emailAddress
+		);
+
+		await usersAndOrganizationsPage.goto();
+
+		await usersAndOrganizationsPage.filterUsers('Unassociated Users');
+
+		await expect(
+			usersAndOrganizationsPage.usersTableCell(user3.alternateName)
+		).toBeVisible();
+		await expect(
+			usersAndOrganizationsPage.usersTableCell('test')
+		).toBeVisible();
+		await expect(
+			usersAndOrganizationsPage.usersTableCell(user1.alternateName)
+		).not.toBeVisible();
+		await expect(
+			usersAndOrganizationsPage.usersTableCell(user2.alternateName)
+		).not.toBeVisible();
+	}
+);
+
+test(
+	'Can assign user group to user',
+	{tag: '@LPD-57819'},
+	async ({apiHelpers, editUserPage, page, usersAndOrganizationsPage}) => {
+		const user = await apiHelpers.headlessAdminUser.postUserAccount();
+		const userGroup = await apiHelpers.headlessAdminUser.postUserGroup();
+
+		await usersAndOrganizationsPage.goto();
+
+		await (
+			await usersAndOrganizationsPage.usersTableRowLink(
+				user.alternateName
+			)
+		).click();
+
+		await editUserPage.membershipsLink.click();
+
+		await expect(editUserPage.membershipsNoUserGroupsMessage).toBeVisible();
+
+		await editUserPage.selectUserGroupsButton.click();
+		await editUserPage.selectUserGroupTable.changeView('table');
+		await editUserPage.selectUserGroupTable.cell(userGroup.name).click();
+
+		await expect(
+			(
+				await editUserPage.membershipsUserGroupsTableRow(
+					0,
+					userGroup.name,
+					true
+				)
+			).row
+		).toBeVisible();
+
+		await editUserPage.saveButton.click();
+
+		await waitForAlert(page);
+
+		await usersAndOrganizationsPage.goto();
+
+		await expect(
+			usersAndOrganizationsPage.usersTableCell(userGroup.name)
+		).toBeVisible();
+	}
+);
+
+test(
+	'Can receive new user notification when single approver workflow is applied to user',
+	{tag: ['@LPD-57819', '@LPS-200153']},
+	async ({apiHelpers, configurationTabPage, notificationsPage, page}) => {
+		await configurationTabPage.goTo();
+
+		await nextPage(page);
+
+		await configurationTabPage.assignWorkflowToAssetType(
+			'Single Approver',
+			'User'
+		);
+
+		await apiHelpers.headlessAdminUser.postUserAccount();
+
+		await notificationsPage.goto();
+
+		await expect(
+			notificationsPage.workflowReviewMessage('User')
+		).toBeVisible();
+
+		await notificationsPage.selectAllItemsCheckbox.check();
+		await notificationsPage.deleteButton.click();
+
+		await waitForAlert(page, 'Notifications were deleted successfully.');
+
+		await configurationTabPage.goTo();
+
+		await nextPage(page);
+
+		await configurationTabPage.unassignWorkflowFromAssetType('User');
```

Comment:
can we move these in the finally? In this way if something goes wrong it reset the workflow